docs/tests/development/post_v1_message-batches/validation.md

```diff
@@ -193,6 +193,72 @@ This test uses the ‘X-Correlation-Id’ header, when provided in a request it
 | 76491414-d0cf-4655-ae20-a4d1368472f3 | Is tested to ensure that when a correlation identifier is sent, we respond with the same value.               |
 
 
+## Scenario: An API consumer submitting a request with an invalid address postcode receives a 400 ‘Invalid Value’ response
+
+A valid contact detail must be structured in this format: { sms: value, email: value, address: { lines: [], postcode: value } }
+
+**Given** the API consumer provides an message body with an invalid postcode
+<br/>
+**When** the request is submitted
+<br/>
+**Then** the response returns a 400 invalid value error
+<br/>
+
+**Asserts**
+- Response returns a 400 ‘Invalid Value’ error
+- Response returns the expected error message body with references to the invalid attribute
+- Response returns the ‘X-Correlation-Id’ header if provided
+
+| Value                    | Description                                                               |
+|--------------------------|---------------------------------------------------------------------------|
+| None                     | Are tested to ensure that null personalisation values are not accepted    |
```

Comment:
I believe these need updating. They aren't the actual values used by the tests.
It's the same for the other occurrences.

Reply:
Ahh yes good spot